charts/nginx-gateway-fabric/values.yaml

```diff
@@ -134,6 +134,29 @@ nginx:
   # -- Is NGINX Plus image being used
   plus: false
 
+  # Configuration for NGINX Plus usage reporting.
+  usage:
+    # -- The name of the Secret containing the JWT for NGINX Plus usage reporting. Must exist in the same namespace
+    # that the NGINX Gateway Fabric control plane is running in (default namespace: nginx-gateway).
+    secretName: "nplus-license"
+
+    # -- The endpoint of the NGINX Plus usage reporting server. Default: product.connect.nginx.com
+    endpoint: ""
```

Comment:
Why not just set the endpoint value to "product.connect.nginx.com" since it's the default?

Reply:
I think I answered my own question as I continued reviewing. The minimal mgmt config doesn't need an endpoint.

Reply:
Yep, this is the nginx default, so didn't want to specify in case that value ever changed. We would still need to update this comment.